src/LndClient/Data/OpenChannel.hs

```diff
@@ -0,0 +1,58 @@
+{-# LANGUAGE DeriveGeneric #-}
+
+module LndClient.Data.OpenChannel
+  ( OpenChannelRequest (..),
+    ChannelPoint (..),
+    ResultWrapper (..),
+  )
+where
+
+import Data.Aeson (FromJSON (..), ToJSON (..))
+import Data.Text (Text)
+import GHC.Generics (Generic)
+import LndClient.Utils (stdParseJSON, stdToJSON)
+
+newtype ResultWrapper a
```

Comment:
This type already exists, let's reuse it (it was in Invoice file, probably should go to separate)

Reply:
Okay, moved to Newtypes